src/http/one/TeChunkedParser.cc

```diff
@@ -115,55 +116,77 @@ Http::One::TeChunkedParser::parseChunkSize(Http1::Tokenizer &tok)
  * ICAP 'use-original-body=N' extension is supported.
  */
 bool
-Http::One::TeChunkedParser::parseChunkExtension(Http1::Tokenizer &tok, bool skipKnown)
+Http::One::TeChunkedParser::parseChunkExtensions(Tokenizer &tok, bool skipKnown)
 {
+    while (!tok.atEnd()) {
+        if (!ParseBws(tok)) // Bug 4492: IBM_HTTP_Server sends SP after chunk-size
+            return false;
+
+        if (skipLineTerminatorIfAny(tok)) {
```

Comment:
It would be possible with adjusting other code. Anyway, since we need to check for BWS and line terminators in a common case (when there are no chunk extensions at all) we would need to duplicate these checks before the chunk-ext loop while keeping them inside of it. I think the current solution is better and keep it as is.